src/rsz/src/RepairSetup.cc

```diff
@@ -1595,13 +1622,16 @@ void RepairSetup::printProgress(const int iteration,
 
   if (start && !end) {
     logger_->report(
-        "Iteration | Removed | Resized | Inserted | Cloned Gates | Pin Swaps |"
+        "Iteration | Removed | Resized | Inserted | Cloned Gates | Pin Swaps "
+        "|"
         "   WNS   |   TNS   | Endpoint");
```

Comment:
There is some odd wrapping here.  I don't see any actual change so I'm confused why this is changed.  Same below.  At least it would be nice to join the string into just two lines.  Do you have some editor setting that is causing this?

Reply:
Fixed